### ISSUE TITLE
Placeholder Text Not Aligned Correctly On Multiple

### ISSUE THREAD

**Issue by Joyrex** [berkan52/select2-bootstrap4-theme#8]
The placeholder text is misaligned when using Select2 Bootstrap 4 theme in multiple (tag) mode:
![image](https://user-images.githubusercontent.com/651646/40334793-7b2fb790-5d25-11e8-997f-662039a494f8.png)
Adjusting the `padding-top` on `.select2-container--bootstrap4 .select2-selection--multiple .select2-selection__rendered` from `.75rem` to `.50rem' aligns it correctly:
![image](https://user-images.githubusercontent.com/651646/40334829-d1a37454-5d25-11e8-89aa-1ff1cb6837cb.png)

This is using Bootstrap 4.1.1 and Select2 Bootstrap4 Theme (latest)

**Comment by VictorioBerra** [comment]
@joyrex thanks! Would you mind submitting a pull request with your fix?

**Comment by Joyrex** [comment]
I've... never done that before! I went to Pull Requests, but it appears that I would have to fork this before I could create the pull request... right?

**Comment by VictorioBerra** [comment]
@Joyrex 

Correct, fork the repo, clone it and make changes and **test your changes**. Especially with CSS make sure your fix did not just break the way other things look. IE: did your change just break the positioning of the little "x" button for tags? Did your change break single select dropdowns? Try and test everything.

When you are happy with it, commit and push your changes and GitHub will show you a "create pull request" button. Its very easy.

Some things to keep in mind:

- Make sure you follow the existing styles, formatting and structure that is in the code base. Make it clean.
